packages/cascara/src/ui/Table/0-Table.mdx

```diff
@@ -40,23 +39,27 @@ const data = {[
 
 ## actions
 
-Actions represent 'verbs' we want to apply to our Table rows, like 'view' to navigate to another page with detailed information about the row.
-Another example would be 'Delete' to delete a row. 'actions' will be appended to each row, they'll appear as buttons.
-By defining an 'action', you are telling Table how you want to be notified when your 'action' button is pressed. Via the 'onAction' event.
+Think of actions as **verbs** you would want to apply to each of the Table rows, like **view** to navigate to another page with detailed information about the row.
+Another example would be **Delete** to delete a row. actions will be appended to each row as buttons.
 
-> Table does not provide the logic to handle 'actions', the implementation for deleting a row, navigating to another page, etc. it's up to you.
+When you define an action, you are telling Table how you want to be notified when your action's button is clicked. Via the **onAction** event.
```

Comment:
Same here. If `onAction` is a prop, we should show it as code and not bold.

Reply:
ok

Reply:
@lordpixel this was marked as resolved, but this still shows up as bold and not as code.

Reply:
my bad, I didn't push my last change, done.